cli/src/commands/start.rs

```diff
@@ -182,6 +182,10 @@ pub async fn start(
                         enabled: false,
                         override_port: None,
                     },
+                    health_details: HealthOverrideSettings {
+                        enabled: true,
```

Comment:
any reason we even want anyone to ever turn it off? feels like something making the yaml file more clunky i would probs add this to the global one https://rindexer.xyz/docs/start-building/yaml-config/global as a single thing aka health_override_port: and then make it always on